packages/v4/src/content/training/tutorials/katacoda-script.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import './tutorials.css'
+
+const KatacodaExample = ({katacodaId}) => {
+  React.useEffect(() => {
+      const script = document.createElement('script');
+      script.setAttribute('src', '//katacoda.com/embed.js');
+      document.body.appendChild(script);
+      const outterContent = document.getElementsByClassName('ws-mdx-content')[0];
+      outterContent.setAttribute('style', 'padding: 0');
+      const innerContent = document.getElementsByClassName('ws-mdx-content-content')[0];
+      innerContent.setAttribute('style', 'max-width: none');
+      return () => {
+        document.body.removeChild(script);
+      }
```

Comment:
Can you wrap this in `if (typeof window !== 'undefined)`? We don't need to this for SSR and it seems to be slowing the build down.